node/mock-answer.ts

```diff
@@ -29,7 +44,7 @@ export class MockAnswers {
         this._answers = answers;
     }
 
-    public getResponse(cmd: string, key: string, debug: (message: string) => void): any {
+    public getResponse(cmd: MockedCommand, key: string, debug: (message: string) => void): any {
```

Comment:
Breaking change? May want to bump at least the minor since it's mock and not the main code.

Reply:
Maybe MockCommand?

Reply:
Yeah, it's technically a breaking change if people are calling `getResponse` directly for some reason.  There aren't any uses right now in vsts-tasks -- it's just used internally in mock-task and mock-toolrunner.

I said `MockedCommand` since it's "one of the commands that are mocked" and not the mock itself.

Reply:
There might be 3rd party extensions using it

Reply:
Yeah, we can bump the minor version